modules/otel-collector-components.adoc

```diff
@@ -82,6 +82,40 @@ The Jaeger receiver ingests traces in the Jaeger formats.
 <4> The Jaeger Thrift Binary endpoint. If omitted, the default `+0.0.0.0:6832+` is used.
 <5> The  server-side TLS configuration. See the OTLP receiver configuration section for more details.
 
+[id="hostmetrics-receiver_{context}"]
+=== Host Metrics Receiver
+
+:FeatureName: The Host Metrics Receiver
+include::snippets/technology-preview.adoc[]
+
+The Host Metrics Receiver ingests metrics in the OTLP format.
+
+.OpenTelemetry Collector custom resource with an enabled Host Metrics Receiver
+[source,yaml]
+----
+  config: |
+    receivers:
+      hostmetrics:
+        collection_interval: 1m # <1>
+        initial_delay: 1s # <2>
+        root_path: / # <3>
+        scrapers: # <4>
+          cpu:
+          memory:
+          disk:
+          filesystem:
+
+    service:
+      pipelines:
+        metrics:
+          receivers: [hostmetrics, hostmetrics/disk]
+
```

Comment:
```suggestion
---
apiVersion: v1
kind: ServiceAccount
metadata:
  name: otel-hostfs-daemonset
  namespace: <namespace>
---
apiVersion: security.openshift.io/v1
kind: SecurityContextConstraints
allowHostDirVolumePlugin: true
allowHostIPC: false
allowHostNetwork: false
allowHostPID: true
allowHostPorts: false
allowPrivilegeEscalation: true
allowPrivilegedContainer: true
allowedCapabilities: null
defaultAddCapabilities:
- SYS_ADMIN
fsGroup:
  type: RunAsAny
groups: []
metadata:
  name: otel-hostmetrics
readOnlyRootFilesystem: true
runAsUser:
  type: RunAsAny
seLinuxContext:
  type: RunAsAny
supplementalGroups:
  type: RunAsAny
users:
- system:serviceaccount:<namespace>:otel-hostfs-daemonset
volumes:
- configMap
- emptyDir
- hostPath
- projected
---
apiVersion: opentelemetry.io/v1alpha1
kind: OpenTelemetryCollector
metadata:
  name: otel
  namespace: <namespace>
spec:
  serviceAccount: otel-hostfs-daemonset
  mode: daemonset
  volumeMounts:
    - mountPath: /hostfs
      name: host
      readOnly: true
  volumes:
    - hostPath:
        path: /
      name: host
  config: |
    receivers:
      hostmetrics:
        collection_interval: 10s # <1>
        initial_delay: 1s # <2>
        root_path: / # <3>
        scrapers: # <4>
          cpu:
          memory:
          disk:
    service:
      pipelines:
        metrics:
          receivers: [hostmetrics]
```

Reply:
I've applied this suggestion and removed the first line `---` from it.